integration_tests/contracts/contracts/Gravity.sol

```diff
@@ -220,7 +220,7 @@ contract Gravity is ReentrancyGuard {
 		uint8[] memory _v,
```

Comment:
does it need to have a copy of Gravity.sol here instead of taking directly from the gravity bridge repository (e.g. a submodule or just a link to the raw file revision)?